addons/sms/static/src/js/fields_sms_widget.js

```diff
@@ -164,6 +164,20 @@ var SmsWidget = FieldTextEmojis.extend({
         this._super.apply(this, arguments);
         this._updateSMSInfo();
     },
+
+    /**
+     * @override
+     * @private
+     */
+    _onBlur: function () {
```

Comment:
Noting that JS reviewers may be picky, it should be set before _onChance (ordered by alphabetical order).